### ISSUE TITLE
BUG - Sev 3 - iOS - Accessibility, unable to select links in content of secure messages

### ISSUE THREAD

**Issue by DJUltraTom** [department-of-veterans-affairs/va-mobile-app#7615]
## What happened?

In the process of creating clickable links in the body of secure messages we uncovered a limitation in react native on iOS that doesn't allow the links to be select when voiceover is on.

## Specs:

- Device: ipad mini
- OS: iOS
- User Account: N/A ( use demo mode for message with links)
- Accessibility State: voiceover enabled

## Steps to Reproduce

- with voiceover on navigate to a secure message with links in the body of the message
- attempt to explore by touch, or swipe to a link
- observe links are not selectable

## Desired behavior

- links should be selectable

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: Low 
- Frequency: Low 

- 3 - Low

## Linked to Story

https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/5712

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the [bug epic](https://app.zenhub.com/workspaces/va-mobile-product-view-610035bc5395bb000e62e529/issues/gh/department-of-veterans-affairs/va-mobile-app/5367)

**Comment by bischoffa** [comment]
@DJUltraTom following up on Therese's question above.

**Comment by Sparowhawk** [comment]
this was fixed, duplicate ticket
